### ISSUE TITLE
Accessibility - Entering text in search doesn't seem to work

### ISSUE THREAD

**Issue by podfeet** [NosillaCast/a-world-clock#13]
I can't enter text in VoiceOver for the search box.  I can get into the box but typing doesn't work.

**Comment by podfeet** [comment]
Fixed somewhere along the lines!
